ui/apps/dashboard/src/layout/index.tsx

```diff
@@ -52,6 +55,16 @@ export const MainLayout: FC = () => {
           <Outlet />
         </AntdContent>
       </AntdLayout>
+      <Navigation onTerminalClick={() => setIsTerminalOpen(true)} />
```

Comment:
here you will add two navigation component ?
![image](https://github.com/user-attachments/assets/eb7a158b-1be2-4cce-b8c2-937ca4855ff8)